app/controllers/tickets_controller.rb

```diff
@@ -20,4 +21,20 @@ def check_load_resource
   def load_tickets
     @tickets = @conference.tickets.visible.order(:title)
   end
+
+  def load_currency_conversions
+    @currency_conversions = @conference.currency_conversions
+    @currencies = [@conference.tickets.first.price_currency]
+    @currencies |= @currency_conversions.map(&:to_currency).uniq
+    @currency_meta = @currencies.map do |currency|
```

Comment:
because the loop is ~10 lines, a new method for that logic may be more clean and readable